### ISSUE TITLE
Update env vars to work with `veda-deploy`

### ISSUE THREAD

**Issue by slesaad** [NASA-IMPACT/veda-data-airflow#112]
# Description

The [last PR for adding workflows-api](https://github.com/NASA-IMPACT/veda-data-airflow/pull/108) to the repo introduced a bunch of new environment variables. Now that we're planning to use `veda-deploy` for deployment, we need to be mindful that dependent env vars have been outputted via CfnOutput from the dependencies and that the format of environment variables is correct.

A lot of these new env vars already are exported from the other projects and have a certain format. Some can be derived from the existing env vars.

We'll need to update those.

Here's the list:
```bash
data_access_role_arn="${DATA_ACCESS_ROLE_ARN}" // VEDA_DATA_ACCESS_ROLE_ARN
workflow_root_path="${WORKFLOW_ROOT_PATH}" // add new in secrets
ingest_url="${INGEST_URL}" // VEDA_STAC_INGESTOR_API_URL
raster_url="${RASTER_URL}" // VEDA_RASTER_URL
stac_url="${STAC_URL}"  // VEDA_STAC_URL
cloudfront_id="${CLOUDFRONT_ID}" // secrets
jwks_url="${JWKS_URL}" // can be derived from VEDA_COGNITO_APP_SECRET
cognito_userpool_id="${COGNITO_USERPOOL_ID}" // can be derived from VEDA_COGNITO_APP_SECRET
cognito_client_id="${COGNITO_CLIENT_ID}"    //  can be derived from VEDA_COGNITO_APP_SECRET
```

**Comment by anayeaye** [comment]
veda-auth notes

These variables should be in the github composite runner environment after veda-deploy has run the deploy backend step. I think it is a problem that these are added outside of the veda-deploy cicd because it is hard to trace. 
`VEDA_CLIENT_ID` of the programmatic user password client used by ingest api
`VEDA_USERPOOL_ID`

**Comment by slesaad** [comment]
closed by #111
